internal/locate/region_request_test.go

```diff
@@ -827,3 +864,20 @@ func (s *testRegionRequestToSingleStoreSuite) TestRegionRequestStats() {
 	}
 	s.Contains(expecteds, access.String())
 }
+
+type noCauseError struct {
```

Comment:
@nolouch 
Unrelated changes are included.

Reply:
cc @cfzjywxk 
should pick https://github.com/tikv/client-go/pull/1344, otherwise, will panic.